akka-kernel/src/main/dist/bin/akka-cluster

```diff
@@ -33,9 +33,13 @@ function get() {
 }
 
 function ensureNodeIsRunningAndAvailable {
-    REPLY=$(get Available) # redirects STDERR to STDOUT before capturing it
+    REPLY=$(get Available 2>&1) # redirects STDERR to STDOUT before capturing it
     if [[ "$REPLY" != *true ]]; then
-        echo "Akka cluster node is not available on $HOST, due to $REPLY"
+        if [[ "$REPLY" == *"Cannot convert result to a string." ]]; then
```

Comment:
That'd have to mean the `Available` field of the `akka/Cluster` MBean would have been changed to something that cannot be converted to a string.

I'm not sure whether that is even possible, but if it is, it'd require code changes to the MBean, and I don't know why we'd make such a backwards-incompatible change.